### ISSUE TITLE
Some WCAG accessibility improvements

### ISSUE THREAD

**Issue by mvdzel** [HL7/ig-template-base2#32]
We received some feedback on the accessibility of our IG and found that some issues cannot be fixed in our template, but need changes to the base template.
Most of the following issues have effect on screen readers and some on readability.

**Minimum color contrast in footer ([WCAG 1.4.3](https://www.w3.org/TR/wcag/#contrast-minimum))**
Suggest to use white as link color in the footer, just as in the menu. This is fixed in out template, but could benefit others.

**Prev, Bottom/Top, and Next Links Use Invalid HTML Structure**
The navigation links are now inside header and footer breadcrumb so screen readers "think" it is part of that. 
Move it out of the breadcrumb and make it a nav-element.
Also make the text complete instead of 'prev' use 'previous' or add `aria-label="Previous page"`
 
**Flag announced twice**
Remove `title` attribute from flag; just keep `alt` for screen readers.

**The Main Page Title Is Not Marked as an H1 (WCAG 2.4.6)**
Why start at H2?

**Decorative Images Have a Period as Alt Text Instead of Being Empty**
In generated heirarchy `alt="."` is read out by screen readers; just make alt empty.

**Comment by lmckenzi** [comment]
We can do most of these things.
Footer color contrast:
- We don't want the link color to be the same as the static text color.
- We will make *all* of the static text (including the stuff on the second line) white, not yellow.  We'll then change the link color to be yellow.

For direction tags, will try to make this work while still retaining current layout position.  The layout position has to be retained.

Flag announced twice - We need to have the title because readers might not recognize the flag and need the flyover to see what it is.  If it works to keep the title and drop the alt, we could do that (if screen readers can't recognize that alt and title are the same and are reading both).  Please confirm if that's an appropriate change.

H2 is something we can't change.  We have hundreds of IGs whose content starts at H3 because the page starts at H2.  Changing this would mean changing a bunch of tools as well as the source for every IG Publisher IG in existence, and that's not going to happen.

We can approve this once we have a final decision on what to do about flag.

**Comment by lmckenzi** [comment]
Based on a web search, it appears that if you have to have title, best practice is to have both alt and title, even if it results in some redundancy.    We will change alt to say "[country] flag" and leave the title saying "[country]".

We'll insert a <main> tag right after the plain language tag to help make it obvious where the actual 'content' of the page is.

We have a few alternatives on <h1>
a) We can change the existing <h2> in the template which is the page title to be an <h1/> instead and then add an an innocuous (and hidden) <h2> that says something like "Page content".  This would at least make the hierarchy correct
b) We can examine whether the publisher can auto-rewrite all of the heading levels to decrement all of them (if necessary) by one level (so that what is <h2> in the template becomes <h1> and what is <h3> in content becomes <h2>, etc.  @grahamegrieve will investigate.  If we do this, we'll control it with a template as it'll need to be accompanied by CSS changes.
